src/components/resume.tsx

```diff
@@ -70,50 +100,88 @@ function Basics(props: Basics) {
 function WorkExperience(props: { experiencies: Work[] }) {
   if (props.experiencies.length === 0) return null;
 
+  let groupedExperiences = props.experiencies.reduce(
+    (accumulator: Work[][], current) => {
```

Comment:
```suggestion
    (works: Work[][], work) => {
```

Reply:
Thanks, I'll be uploading that with my new changes